packages/api/src/module/read/course-progress/course-progress.read-module.ts

```diff
@@ -25,4 +26,18 @@ export class CourseProgressReadModule {
       },
     });
   }
+
+  @OnEvent('LearningMaterialTasks.TaskWasCompleted')
```

Comment:
On MIRO we have 'LearningMaterial**s**Tasks'
We need to keep it with the same value for @OnEvent to work. 
If you think that it's better name please change it in MIRO and related tasks. 

Good job! I haven't known that such increment is possible. Are you sure it's for relation dbs too? 
It'd be nice to test it, but we wait for merge of this PR with test example first:
https://github.com/CodersCrew/coderscamp/pull/237

Please make a review :)

Reply:
name corrected, 
Info about update I found here https://www.prisma.io/docs/concepts/components/prisma-client/crud#update-a-number-field
and I couldn't use sth like "learningMaterialsCompletedTasks: learningMaterialsCompletedTasks + 1", so I think increment should work